app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/BakeManifestStageForm.tsx

```diff
@@ -13,89 +14,22 @@ interface IBakeManifestStageFormProps {
 export class BakeManifestStageForm extends React.Component<
   IBakeManifestStageFormProps & IFormikStageConfigInjectedProps
 > {
-  public readonly templateRenderers = ['HELM2'];
+  public HELM_RENDERER = 'HELM2';
+  public KUSTOMIZE_RENDERER = 'KUSTOMIZE';
+  // public readonly templateRenderers = [this.HELM_RENDERER, this.KUSTOMIZE_RENDERER];
```

Comment:
guessing we will uncomment this when we remove the feature flag?

Reply:
Nice catch. I meant to remove that, so I'll pull it out. It'll make another appearance once the feature flag is removed. :)